test/tools/libtesteth/TestSuite.cpp

```diff
@@ -122,26 +122,29 @@ void TestSuite::runAllTestsInFolder(string const& _testFolder) const
 {
 	// check that destination folder test files has according Filler file in src folder
 	string const filter = test::Options::get().singleTestName.empty() ? string() : test::Options::get().singleTestName;
-	vector<fs::path> const compiledFiles = test::getJsonFiles(getFullPath(_testFolder), filter);
+	vector<fs::path> const compiledFiles = test::getFiles(getFullPath(_testFolder), ".json|.yaml" ,filter);
 	for (auto const& file: compiledFiles)
 	{
 		fs::path const expectedFillerName = getFullPathFiller(_testFolder) / fs::path(file.stem().string() + c_fillerPostf + ".json");
+		fs::path const expectedFillerName2 = getFullPathFiller(_testFolder) / fs::path(file.stem().string() + c_fillerPostf + ".yaml");
```

Comment:
So you have the magic `".json|.yaml"` but here you hard-coded the extensions?

Reply:
the function could be used for more general perpuses 
hardcode is bad.